package.json

```diff
@@ -36,7 +36,7 @@
     "babel-preset-stage-1": "^6.22.0",
     "conventional-changelog": "^2.0.3",
     "del": "^3.0.0",
-    "gulp": "^4.0.0",
+    "gulp": "^3.9.1",
```

Comment:
but we can run those tasks with version 4. isn't it?

Reply:
Hi Kapil, the gulp tasks were written with the 3.x syntax which fails with gulp 4. Since I was trying to minimize the changes, I rolled back the gulp version to 3.x instead of rewriting tasks in the 4.x syntax. Please see a related issue here: https://github.com/froala/aurelia-froala-editor/issues/60#issue-513839863